Makefile

```diff
@@ -19,10 +19,10 @@
 
 # values used in workspace-status.sh
 
-DOCKER_REGISTRY?=cockroachdb
+DOCKER_REGISTRY?=us.gcr.io/chris-love-operator-playground
```

Comment:
Can you just override this on the command line, instead of changing it back all the time?

Reply:
We need to start overriding both in the Makefile and in the examples, as we're getting closer to shipping and something like this could accidentally break things for others